src/ResourceManager/Sql/Commands.Sql/help/Add-AzureRmSqlManagedInstanceKeyVaultKey.md

```diff
@@ -0,0 +1,146 @@
+---
+external help file: Microsoft.Azure.Commands.Sql.dll-Help.xml
+Module Name: AzureRM.Sql
+online version: https://docs.microsoft.com/en-us/powershell/module/azurerm.sql/Add-AzureRmSqlManagedInstanceKeyVaultKey
+schema: 2.0.0
+---
+
+# Add-AzureRmSqlManagedInstanceKeyVaultKey
+
+## SYNOPSIS
+Adds a key vault key to the provided Managed Instance. 
+## SYNTAX
+
+```
+Add-AzureRmSqlManagedInstanceKeyVaultKey [-ResourceGroupName] <String> [-ManagedInstanceName] <String>
```

Comment:
This look like it's out of date because there's only one parameter set - can you generate all these files again?